Lecture2/Task2/TaskTwoMain.java

```diff
@@ -0,0 +1,74 @@
+import java.util.*;
+
+public class TaskTwoMain {
+	public static void main(String[] args) {
+		final String FIRST_LETTER = "M";
+		List<Student> students = new ArrayList<>();
```

Comment:
create factory class to generate list of students 
`StudentFactory.getStudents(int amount)`

Reply:
Will be implemented in the scope of Lecture 3 Hometask